gematria/datasets/pipelines/benchmark_bbs.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2024 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+from absl import app
+from absl import flags
+import apache_beam as beam
+from apache_beam.options import pipeline_options
+
+from gematria.datasets.pipelines import benchmark_bbs_lib
+
+_INPUT_FILE_PATTERN = flags.DEFINE_string(
+    'input_file_pattern',
+    None,
+    'The input file pattern to load annotated blocks from.',
+    required=True,
+)
+_OUTPUT_FILE_PATTERN = flags.DEFINE_string(
+    'output_file_pattern', None, 'The output file path/pattern.', required=True
+)
+
+
+def main(argv) -> None:
+  del argv  # Unused.
```

Comment:
The latest and greatest way of handling unused `argv` is to raise [`app.UsageError`](https://github.com/abseil/abseil-py/blob/0c02cb772ce28ae3f75298e6495657b36b9eee44/absl/app.py#L77) when this happens.